composer.json

```diff
@@ -9,6 +9,7 @@
   ],
   "require": {
     "php": ">=5.5.9",
+    "consolidation/Robo": "dev-master",
```

Comment:
I am using a couple of minor adjustments to the Runner class in Robo not yet in a tagged RC release that make it easier to re-use Runner in other applications.  Robo is currently at 1.0.0-RC1; there should be an RC2 and a 1.0.0 stable shortly.